x-pack/legacy/plugins/siem/public/components/navigation/tab_navigation/index.tsx

```diff
@@ -48,8 +50,19 @@ const navTabs: NavTab[] = [
     disabled: false,
   },
 ];
+// SIDE EFFECT: the following `injectGlobal` overrides the default styling
+// of euiLink because it's implemented as a Tab,
+// eslint-disable-next-line no-unused-expressions
+injectGlobal`
```

Comment:
@XavierM not sure if this is the right approach? both with `injectGlobal` and the `EuiLink` color prop getting handled in `state`

Reply:
I think that we can do it like this without any state or injectGlobal
```
const MyEuiTab = styled(EuiTab)`
  .euiLink {
    color: inherit !important;
    background-color: inherit !important;
  }
  .euiLink.euiLink--primary:focus {
    outline: none !important;
  }
`;
```